### DISCUSSION TITLE
Copyright Question

### DISCUSSION THREAD

**Discussion by FalconinVoid** [nonamecrackers2/crackers-wither-storm-mod#1629] [Q&A] [answered]
Can I use the Mod Icon of CWSM so I can make a mix out of the Icon from Command Block Ascension and CWSM?
Edit: Can I use all things in the assets folder (without the Witherstorm Theme of course) and make remixes out of it? Would that be okay?

**Comment by nonamecrackers2** [comment, marked as answer]
I'd rather if you were to not use the mod icon. But feel free to remix the textures, just don't use them as-is
